src/Settings.hs

```diff
@@ -114,3 +114,8 @@ programPath context@Context {..} = do
         install <- (\l -> l == stage || package == ghc) <$> maybeLatest
         let path = if install then installPath package else buildPath context
         return $ path -/- programName context <.> exe
+
+destDir :: Action FilePath
```

Comment:
Regarding tracking -- it is not necessary now, whether or not `DESTDIR` is re-defined is not important, since each time `install` gets invoked, the whole process will get repeated, using the newest `DESTDIR`